Makefile

```diff
@@ -51,13 +51,13 @@ endif
 	touch $@
 
 $(linux_wrkdir)/.config: $(linux_srcdir)
-	cd $(linux_srcdir); git stash; git apply $(linux_patch)
+	cd $(linux_srcdir); git stash
 	mkdir -p $(dir $@)
 	cp -p $(linux_defconfig) $@
 	$(MAKE) -C $< O=$(linux_wrkdir) ARCH=riscv olddefconfig
 
 $(vmlinux): $(linux_srcdir) $(linux_wrkdir)/.config $(buildroot_sysroot_stamp)
-	$(MAKE) -j -C $< O=$(linux_wrkdir) \
+	echo "n" | $(MAKE) -j 4 -C $< O=$(linux_wrkdir) \
```

Comment:
what's the echo "n" for? Is there a config option we could pass instead?

I also prefer to rely on $(MAKE) to pass along -j from the toplevel rather than hardcoding it. Otherwise the jobserver can get disabled

Reply:
The echo "n" is for the GCC plugins configurations. I will get the following prompt if I remove it.

![Screenshot from 2022-02-03 13-05-29](https://user-images.githubusercontent.com/32671969/152429216-44b553af-54ff-456b-98f5-db39f4ee46dd.png)

I have tried several times to try to include the correct GCC plugins configurations in our linux_defconfig, but I can never get rid of that prompt, so I use echo "n" to disable it (by saying no) for now.

Reply:
kk fine for now. Seems odd though, maybe there's some magical invocation that doesn't need it